clients/drcachesim/analyzer_multi.cpp

```diff
@@ -143,7 +143,8 @@ analyzer_multi_t::analyzer_multi_t()
         }
     } else {
         // Legacy file.
-        if (!init_scheduler(op_infile.get_value(), op_verbose.get_value()))
+        if (!init_scheduler(op_infile.get_value(),
+            INVALID_THREAD_ID /*all threads*/, op_verbose.get_value()))
```

Comment:
As the tests show this fails clang-format: see https://github.com/DynamoRIO/dynamorio/actions/runs/5009310064/jobs/8978112282?pr=6052